subprojects/distributions/src/changes/accepted-public-api-changes.json

```diff
@@ -1,3 +1,14 @@
 {
-    "acceptedApiChanges": []
+    "acceptedApiChanges": [
+        {
+            "type": "org.gradle.caching.local.DirectoryBuildCache",
```

Comment:
The check is not complaining about the new methods, but about the missing `@since` annotations. We should add that annotations and it should not show up in `accepted-public-api-changes`. The whole class is `Incubating` after all, right?

Reply:
Ah, I was wondering why `@Incubating` was not taken into account...